src/ezsp/lib-ezsp-main.h

```diff
@@ -52,23 +49,23 @@ class CLibEzspMain : public CEzspDongleObserver, CGpObserver
      * @param timerbuilder An ITimerFactory used to generate ITimer objects
      * @param requestZbNetworkResetToChannel Set this to non 0 if we should destroy any pre-existing Zigbee network in the EZSP adapter and recreate a new Zigbee network on the specified 802.15.4 channel number
      */
-    CLibEzspMain(IUartDriver* uartDriver, TimerBuilder &timerbuilder, unsigned int requestZbNetworkResetToChannel=0);
+    CLibEzspMain( NSSPI::IUartDriver* uartDriver, NSSPI::TimerBuilder &timerbuilder, unsigned int requestZbNetworkResetToChannel);
 
     CLibEzspMain() = delete; /*<! Construction without arguments is not allowed */
     CLibEzspMain(const CLibEzspMain&) = delete; /*<! No copy construction allowed */
-    CLibEzspMain& operator=(CLibEzspMain) = delete; /*<! No assignment allowed */    
+    CLibEzspMain& operator=(CLibEzspMain) = delete; /*<! No assignment allowed */
 
     /**
      * @brief Register callback on current library state
      *
      * @param newObsStateCallback A callback function of type void func(CLibEzspState i_state), that will be invoked each time our internal state will change (or nullptr to disable callbacks)
      */
-    void registerLibraryStateCallback(FStateCallback newObsStateCallback);
+    void registerLibraryStateCallback(FLibStateCallback newObsStateCallback);
 
     /**
-     * @brief Register callback to receive all authenticated incoming green power frames
+     * @brief Register callback to receive all incoming greenpower sourceId
      *
-     * @param newObsGPFrameRecvCallback A callback function that will be invoked each time a new valid green power frame is received from a known source ID (or nullptr to disable callbacks)
+     * @param newObsGPFrameRecvCallback A callback function of type void func(CGpFrame &i_gpf), that will be invoked each time a new valid green power frame is received from a known source ID (or nullptr to disable callbacks)
      */
```

Comment:
Seems to try to revert https://github.com/Legrandgroup/libezsp/commit/ed5706ac038092d4561bec8fd615e27c9b954dca ?

Reply:
I don't understand your question ?

Reply:
The change in comments seems to do the inverse of what was submitted in ed5706a.